minifi/minifi-docs/src/main/markdown/minifi-java-agent-quick-start.md

```diff
@@ -106,9 +108,48 @@ config.sh transform input_file output_file
 
 **Note:** You can use one template at a time, per MiNiFi instance.
 
-
 **Result:** Once you have your _config.yml_ file in the `minifi/conf` directory, launch that instance of MiNiFi and your dataflow begins automatically.
 
+### Utilizing a C2 Server via the c2 protocol
+If you have a [C2 server](../../../../minifi-c2/README.md) running you can expose the whole config.yml for the agent to download. As the agent is heartbeating via the C2 protocol changes in flow version will trigger automatic config updates.
+
+1. Launch C2 server
+2. Configure MiNiFi for C2 capability
+```
+c2.enable=true
+c2.config.directory=./conf
+c2.runtime.manifest.identifier=minifi
+c2.runtime.type=minifi-java
+c2.rest.url=http://localhost:10090/c2/config/heartbeat
+c2.rest.url.ack=http://localhost:10090/c2/config/acknowledge
+c2.agent.heartbeat.period=5000
+#(Optional) c2.agent.identifier=123-456-789
+c2.agent.class=agentClassName
+```
+3. Configure MiNiFi to recognise config.yml changes
```

Comment:
I think "recognize" is the more common spelling. Italicize config.yml:

Configure MiNiFi to recognize _config.yml_ changes

Reply:
Thanks, there are a few words were "z" vs "s" is basically British vs US spelling. I'm ok with either :) Changed.